testing/configs/issue-1.yaml

```diff
@@ -2,4 +2,8 @@ output: testing/output/issue-1-output.json
 custom_transform: customtransforms/custom_single_json_retriever.py
 
 file_groups:
-    issue_1: ["testing/resources/browsertime.json"]
+    issue_1:
+        [
+            "testing/resources/browsertime.json",
+            "testing/resources/browsertime\ -\ Copy.json",
```

Comment:
Can you remove the spaces in the name? I think `browsertime_copy.json` would work best.